backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsLogManager.java

```diff
@@ -0,0 +1,91 @@
+package backtraceio.library.breadcrumbs;
+
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.Map;
+
+import backtraceio.library.enums.BacktraceBreadcrumbLevel;
+import backtraceio.library.enums.BacktraceBreadcrumbType;
+import backtraceio.library.logger.BacktraceLogger;
+
+public class BacktraceBreadcrumbsLogManager {
+
+    private String breadcrumbLogPath;
+
+    private final String LOG_TAG = BacktraceBreadcrumbsLogManager.class.getSimpleName();
+
+    private long breadcrumbId = 0;
+
+    private BacktraceQueueFileHelper backtraceQueueFileHelper;
+
+    /**
+     * We truncate messages longer than this
+     */
+    private final int maxMessageSizeBytes = 1024;
+
+    /**
+     * We stop adding new attributes once we hit this limit
+     */
+    private final int maxAttributeSizeBytes = 1024;
+
+    public BacktraceBreadcrumbsLogManager(String breadcrumbLogPath, int maxQueueFileSizeBytes) throws IOException, NoSuchMethodException {
+        this.breadcrumbLogPath = breadcrumbLogPath;
```

Comment:
nit:
do we need at all breadcrumbLogPath here in this class?

Reply:
Looks like we don't need this at all, good catch!